scale/docs/rest/job.rst

```diff
@@ -1050,3 +1050,120 @@ These services provide access to information about "all", "currently running" an
++=========================================================================================================================+
+| Returns detailed information about input files associated with a given Job ID.                                          |
++-------------------------------------------------------------------------------------------------------------------------+
+| **GET** /jobs/{id}/input_files/                                                                                         |
+|         Where {id} is the unique identifier of an existing job.                                                         |
++-------------------------------------------------------------------------------------------------------------------------+
+| **Query Parameters**                                                                                                    |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| started            | ISO-8601 Datetime | Optional | The start of the time range to query.                               |
+|                    |                   |          | Supports the ISO-8601 date/time format, (ex: 2015-01-01T00:00:00Z). |
+|                    |                   |          | Supports the ISO-8601 duration format, (ex: PT3H0M0S).              |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| ended              | ISO-8601 Datetime | Optional | The end of the time range to query.                                 |
+|                    |                   |          | Supports the ISO-8601 date/time format, (ex: 2015-01-01T00:00:00Z). |
+|                    |                   |          | Supports the ISO-8601 duration format, (ex: PT3H0M0S).              |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| time_field         | String            | Optional | The time series that the given time range is applied to.            |
+|                    |                   |          | Valid fields: 'source','data', and 'last_modified'                  |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| file_name          | String            | Optional | The name of a specific file to be returned.                         |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
+| job_input          | String            | Optional | The `input_data` entry name in the `job_data` JSON definition       |
+|                    |                   |          | of the job                                                          |
++--------------------+-------------------+----------+---------------------------------------------------------------------+
```

Comment:
Couple of tweaks to apply to all of the new API docs:
1. For time_field, list the three possible values and include a sentence describing what each one means. The sources API doc has an example of this.
2. For file_name, change the description to something like "Returns only input files with this file name".
3. For job_input/recipe_input, change the description to something like "Returns files for this job/recipe input".